Source/SpeechToTextV1/SpeechToTextSession.swift

```diff
@@ -119,7 +133,7 @@ public class SpeechToTextSession {
      
      If set, the `onConnect()` callback will be invoked after the session connects to the service.
      */
-    public func connect() {
+    public func connect() { 
```

Comment:
Hah! Interesting to see this change. I think we had added some code to `connect()` then removed it in favor of the lazy var. Must have missed the space. It really doesn't matter, but now that I've seen it I can't unsee it. ;)

I'll go ahead and remove this space after finishing the review.